packages/helm-charts/blockscout/scripts/grafana-hook.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+echo Creating Grafana tag
+export CELO_ENV=$CELO_ENV
+export SUFFIX=$SUFFIX
+
+GRAFANA_API_TOKEN=$(echo $GRAFANA_CLOUD_SECRET | jq -r '.grafana_api_token')
+GRAFANA_API_ENDPOINT=$(echo $GRAFANA_CLOUD_SECRET | jq -r '.grafana_endpoint')
+
+GCLOUD_ACCOUNT=$(gcloud config get-value account)
```

Comment:
This won't work in the context of this hook I think, perhaps we should pass the entire string as a parameter.